dev/test/gapic_firestore_v1.ts

```diff
@@ -482,7 +482,9 @@ describe('v1.FirestoreClient', () => {
         ['document', 'name']
       );
       request.document.name = defaultValue1;
-      const expectedHeaderRequestParams = `document.name=${defaultValue1 ?? ''}`;
+      const expectedHeaderRequestParams = `document.name=${
```

Comment:
I assume the changes in these 3 files are all format changes, correct?

Reply:
correct. good question